packages/xod-client/src/core/styles/components/Pin.scss

```diff
@@ -83,3 +83,10 @@
     }
   }
 }
+
+.PinDots {
+  fill: $chalk;
+  font-weight: 300;
+  font-family: $font-family-normal;
+  font-size: $font-size-m;
```

Comment:
```css
font-size: 14px;
font-weight: 500;
```

gave a slightly better result on my PC. The ellipsis is not a text IMO, so it can be styled out-of-band.